src/components/BusinessesList/index.jsx

```diff
@@ -0,0 +1,25 @@
+import React, { useEffect, useState } from "react";
+import http from "../../services/httpService";
+import BusinessCard from "../BusinessCard";
+
+const endPointUrl = `https://api.myjson.com/bins/wcdjs`;
+export default function BusinessesList() {
+  const [businesses, setBusinesses] = useState([]);
+
+  useEffect(() => {
+    (async function getBuisinesses() {
+      const { data } = await http.get(endPointUrl);
```

Comment:
also in case there is an error this will crash the frontend, so make sure to wrap `async await` with `try catch`

Reply:
done